ci-tests/wake_scala_compilation

```diff
@@ -34,5 +34,5 @@ echo "Compile Scala"
 
 export WAKE_PATH=$PATH
 wake --init .
-wake --no-tty -p1 -dv 'compileScalaModule rocketchipScalaModule | getPathResult'
+wake -x 'compileScalaModule rocketchipScalaModule | getPathResult'
```

Comment:
Is there a reason you got rid of the `--no-tty`? I think this affects the carriage return thing for "progress bars", and in CI, when you don't really have a real TTY, it often ends up creating a newline for each carriage return as well.

Reply:
Wake does not need to be told there is no tty. It auto-detects this.
The only reason to use --no-tty is if you actually DO have a tty, but don't want the bards.

Reply:
Ah, I see that you're right. I took a quick look at the actual [CI console log from this PR](https://github.com/chipsalliance/rocket-chip/pull/2584/checks?check_run_id=946222730) and saw some progress bars, but after taking a closer look, I see that they're actually from `curl`.